lib/fastly_nsq/version.rb

```diff
@@ -1,3 +1,3 @@
 module FastlyNsq
-  VERSION = '0.2.3'
+  VERSION = '0.3.0'
```

Comment:
This feels like a big enough change that it warrants a minor version bump, I assume you concur - I'm happy to discuss if otherwise.